0003-messaging-ipc/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: Messaging IPC
+shortname: 3/MIPC
+status: raw
+contributors: Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
```

Comment:
Honestly, I don't see a lot of value in specifying people / names in the doc. Everyone can read the commit history and figure things out.